resources/views/components/menu/account.blade.php

```diff
@@ -1,49 +1,78 @@
-        </li>
+        </li>--}}
     </ul>
-@endguest--}}
+@endguest
 @auth
-    <ul class="navbar-nav">
-        {{--
-        Note: keep icon avatar in its own navbar-nav
-        otherwise it jumps around vertically with other text nav items in the same group
-        --}}
-        <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right">
-            <x-slot name="trigger">
-                <x-user.avatar :user="request()->user()" display="icon" iconSize="sm" :link="false" :tooltip="false" />
-            </x-slot>
-            {{--<x-dropdown-header>{{ request()->user()->username }}</x-dropdown-header>--}}
-            {{--<div class="dropdown-divider"></div>--}}
-            {{--<x-dropdown-item :link="route('user.show', request()->user())">{{ __res('profile', 1) }}</x-dropdown-item>--}}
-            {{--<x-dropdown-item :link="route('friend.index')">{{ __res('friend') }}</x-dropdown-item>--}}
-            {{--<x-dropdown-item :link="route('inbox')">{{ __('Inbox') }}</x-dropdown-item>--}}
-            {{--<a class="dropdown-item" href="{{ route('history.index') }}">History</a>--}}
-            {{--<div class="dropdown-divider"></div>--}}
-            {{--<x-dropdown-item :link="route('settings')">{{ __res('setting') }}</x-dropdown-item>--}}
-            {{--<x-form :action="route('logout')">
-                <button class="dropdown-item">{{ __('Sign Out') }}</button>
-            </x-form>--}}
-        </x-nav-dropdown>
-    </ul>
-    {{--<ul class="navbar-nav">
-        <li class="nav-item hidden lg:inline-block"><span class="nav-link">{{ request()->user()->points_total }}</span></li>
-    </ul>--}}
+    <div class="nav-link flex-col justify-center items-end text-2xs" style="line-height: 1.1em">
+        @if($user->points_total)
+            <div class="text-color cursor-help" title="Points earned in hardcore mode">{{ localized_number($user->points_total) }}</div>
+        @endif
+        @if($user->points_weighted_total)
+            <x-points-weighted-container>
+                <span class='trueratio'>{{ localized_number($user->points_weighted_total) }}</span>
+            </x-points-weighted-container>
+        @endif
+        @if($user->RASoftcorePoints)
+            <div class='softcore cursor-help' title="Points earned in softcore mode">{{ localized_number($user->RASoftcorePoints) }}</div>
+        @endif
+    </div>
+    <x-nav-dropdown trigger-class="py-0" dropdown-class="dropdown-menu-right">
+        <x-slot name="trigger">
+            <x-user.avatar :user="$user" display="icon" iconSize="sm" :link="false" :tooltip="false"/>
+        </x-slot>
+        <x-dropdown-header>{{ $user->username }}</x-dropdown-header>
+        <div class="dropdown-divider"></div>
+        <x-dropdown-item :link="route('user.show', $user)">{{ __res('profile', 1) }}</x-dropdown-item>
+        @if($user->ContribCount > 0 || $user->Permissions >= Permissions::JuniorDeveloper)
+            <div class="dropdown-divider"></div>
```

Comment:
Extra divider if I set my permissions to Registered:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/f7130bab-717b-45ff-ad31-02741235424b)

Why is this dependent on ContribCount?

I think you mistranslated this logic: https://github.com/RetroAchievements/RAWeb/blob/6baf30005587144bacebfadcb246c691360e7f3b/app/Helpers/render/layout.php#L193-L200